src/extension.ts

```diff
@@ -486,6 +486,9 @@ export async function deactivate() {
 
 	tearDown()
 
+	// Clean up file logging resources
+	Logger.cleanup()
```

Comment:
I'm a bit meh about shutdown code. In my experience it tends to be poorly tested, slow, and often the OS' resource management is enough anyway.

In this specific case, in theory you *do* need to flush the stream but you're calling `end` without waiting for the finish event. The only thing making this work would be racing with other slow shutdown code.

In practice, why not wrap the stream in something that looks for `\n` and flushes after that? And then just drop the whole thing on the floor.